installer/tracex_installer_release.iss

```diff
@@ -55,7 +55,7 @@ Source: "src\MsixPackageProject\TraceFiles\demo_threadx.trx"; DestDir: "{app}/Tr
 
 [Icons]
 Name: "{group}\TraceX 6.4\TraceX"; Filename: "{app}\TraceX.exe"
-Name: "{group}\TraceX 6.4\TraceX User's Guide"; Filename: "https://aka.ms/azrtos-tracex-user-guide"
+Name: "{group}\TraceX 6.4\TraceX User's Guide"; Filename: "https://github.com/eclipse-threadx/rtos-docs"
```

Comment:
I think this point directly to:
https://github.com/eclipse-threadx/rtos-docs/blob/main/rtos-docs/tracex/overview-tracex.md

Reply:
updated